app/presenters/payment_presenter.rb

```diff
@@ -0,0 +1,18 @@
+class PaymentPresenter < Rectify::Presenter
+  attribute :params
+  attribute :current_order
+  attribute :form
+  STEP = :payment
+
+  def payment_card
+    form || CreditCardForm.new
+  end
+
+  def current_card(type)
+    current_order.credit_card[type] if current_order.credit_card
```

Comment:
`if current_order.credit_card` is unnecessary

In View use current_card&.some_attribute